### ISSUE TITLE
Stream ID

### ISSUE THREAD

**Issue by boucadair** [danwing/cidfi#41]
https://github.com/danwing/cidfi/blob/8e3eef44bd73ad95e96fc311fd447f53f21774cf/draft-wing-cidfi.md?plain=1#L581

We may consider defining a new QUIC frame, similar in processing to the DATAGRAM frame (but no user data is allowed inside). The frame can be sent in both directions h2n and n2h. No stream ID would be required then.

**Comment by danwing** [comment]
Sounds like a great idea.  One note:  this communication is merely h2h, as no network element is involved -- although the data inside is intended to be relayed to the CIDFI Network Element.  I don't know how to express that in concise words, however.

**Comment by boucadair** [comment]
You are right that the frame is mainly between the ultimate endpoints of a communication.
